gnomad_qc/example_notebooks/ancestry_classification_using_gnomad_rf.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Reduce variant data to only needed annotations to be
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/gnomad_qc/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>